### PR TITLE
Move ImJasonH/gcping to web/

### PR DESCRIPTION
**Pull request by ImJasonH**
This PR wholesale moves the repo at https://github.com/ImJasonH/gcping (except `LICENSE`) into a subdirectory here, called `web/`. It also updates the link in index.html to point to this repo. I _think_ that's the only reference to that GH repo, let me know if you find others.

I did petition Google for personal ownership of gcping, but honestly it probably belongs in this org and I'm happy to donate it back. If you'll have it. :) 

If this is acceptable to you, after this I'd like to:
* update gcping.com to point to this repo
* archive and forward ImJasonH/gcping to this repo
* maybe refactor READMEs together somehow to avoid duplication?
* maybe share config.js and descs.js between the CLI and UI somehow?
* maybe release the CLI tool in `setup.sh`?

(I've gotten to the point where every time there's a new region launched, I just run `setup.sh` and commit the generated result. Improvements to that process gladly welcome.)

### CODE REVIEW

File: web/regen.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package main
```
Comment (rakyll) [inline web/regen.go]: Can you add the license headers?

File: web/config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (rakyll) [inline web/config.js]: Can we generate a Go file in the future so we don't have to embed the endpoints in the command line tool?

Comment (ImJasonH) [reply web/config.js]: Yeah absolutely!

**Comment by ImJasonH** [comment]
This seems to have rotted. Sorry about that 😞 

I'll make some time to spruce it up and send a new PR.

**Pull request closed without merging**